logger.go

```diff
@@ -34,15 +33,10 @@ type Shuttle struct {
 
 // NewShuttle creates a shuttle client from a Config.
 func NewShuttle(cfg *LoggerConfig) (*Shuttle, error) {
-	token, ok := cfg.URL.User.Password()
-	if !ok {
-		return nil, errors.New("no logplex password provided")
-	}
-
 	// Create shuttle config.
 	conf := shuttle.NewConfig()
 	conf.LogsURL = cfg.URL.String()
-	conf.Appname = token
+	conf.Appname = "heroku"
```

Comment:
> We do log as `app`

Always, or should we make this configurable?

> This seems like a small change that hides other problems. 

Happy to hear other suggestions how to solve for this. We could switch on the hostname of the URL or something to know if it is a Logplex URL or not, but not sure there's any benefit to doing that because we'd still end up hardcoding it for the custom URL case, and potentially risk leaking passwords into log streams. 

Another alternative is going back to the drawing board and making all (not just unpeered) add-on providers go through Logplex. We'd need to assess the security of that though.